tests/unit/HelperTest.php

```diff
@@ -48,7 +55,12 @@ public function array_parameter_call_set_method_into_store()
 }
 
 if (!function_exists('app')) {
```

Comment:
No longer need this as composer.json includes the whole framework:

https://github.com/laravel/framework/blob/5.6/composer.json#L90

https://github.com/laravel/framework/blob/5.6/src/Illuminate/Foundation/helpers.php#L104

Reply:
Not sure why php 5.4 & 5.5 are failing. It looks like they are not installing Illuminate packages because their php version requirement?

Reply:
@bweston92 any idea why is it failing?